README.md

```diff
@@ -1,65 +1,63 @@
-# Prometheus Exporter Snap for Hardware Observer charm
-
-
-You can learn more about Hardware Observer on [Charmhub](https://charmhub.io/hardware-observer).
-## Getting Started
-
-Install the snap from snap store:
-
-```bash
-$ sudo snap install prometheus-hardware-exporter --classic
-```
-
-Start the exporter:
-
-```bash
-$ sudo snap start prometheus-hardware-exporter
+# Prometheus Hardware Exporter
+
+Prometheus Hardware Exporter exports Prometheus metrics from BMCs (Baseboard
+management controllers) using IPMI (Intelligent Platform Management Interface)
+and Redfish protocol. It also exports and various SAS (Serial Attached SCSI) and
+RAID (redundant array of inexpensive disks) controllers.
+
+Note that the exporter does not bundle the required third party or proprietary
+software. If you would like to use this exporter you will need to install them
+by yourself. Therefore, Prometheus Hardware Exporter is recommended for
+[Juju](https://juju.is/) users, you can learn more about Hardware Observer on
+[Charmhub](https://charmhub.io/hardware-observer).
+
+## Installation
+
+This package is not published in anywhere yet. For now, you can should install
+this application locally (or via a virtual environment). To install it on a
+virtual environment, you can run:
+
+```shell
+# Install virtualvenv and source it
+pip3 install virtualenv
+python3 -m venv venv
+source venv/bin/activate
+
+# Install the prometheus-hardware-exporter package; the "(venv)" means you are
+# under the virtual environment
+(venv) pip3 install .
 ```
 
-## Snap Configuration
+To verify you have install the package, you can run:
```

Comment:
```suggestion
To verify that the package has been installed successfully, run:
```
I noticed that in some places we're referring to it as a "package" and some other places it's an "application". I don't mind the usage of either term as long as it's consistently used everywhere :smile:

Reply:
Done